app/api/v1/endpoints/trading.py

```
@@ -559,40 +608,52 @@ async def get_recent_trades(
         user_id=str(current_user.id)
     )
     try:
-        # In a real application, this data would come from the database
-        mock_recent_trades = [
+        # Get real trades from database
+        recent_trades_query = db.query(Trade).filter(
```

Ruff (0.12.2)

Undefined name `db`

(F821)